lib/Styles/Gtk/Button.scss

```diff
@@ -3,6 +3,9 @@ button {
     // the necessary SVGs for light/dark and accent color combinations is
     // untenable. Ideally we'd deprecate this in favor of something like a
     // Granite.BackButton with custom drawing; until then, stick an icon in it.
+    @include control;
+    @include border-interactive-roundrect;
```

Comment:
Can we scope this down to `&.textbutton`? That's something I wanted to not repeat here is that image buttons never have this kind of raised style and we have to undo all the default styles for image buttons :sweat_smile: 

Also this looks like it got inserted under the comment for the `.back-button` class :grimacing: